app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -181,6 +181,12 @@ def supported_documents
                 pdf: PdfFiller::Ny213Pdf,
                 include: receiving_213_credit
               },
+              {
+                xml: SubmissionBuilder::Ty2022::States::Ny::Documents::It213,
+                pdf: PdfFiller::Ny213AttPdf,
+                include: @submission.data_source.dependents.select(&:eligible_for_child_tax_credit).length > 6,
```

Comment:
the 6 could be a public constant here (or somewhere else sensible), used in both places in this file and then either passed to or accessed by ny213_att_pdf.rb (and I think the param default should be removed there in either case).